### ISSUE TITLE
Installation through scripts is broken for v1.26.0

### ISSUE THREAD

**Issue by kashifest** [cri-o/cri-o#6477]
### What happened?

```
curl [https://raw.githubusercontent.com/cri-o/cri-o/v1.26.0/scripts/get](https://raw.githubusercontent.com/cri-o/cri-o/v1.26.0/scripts/get%1B[0m)
 % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0Welcome to the CRI-O install script!
Using version: v1.26.0
100  8566  100  8566    0     0  20541      0 --:--:-- --:--:-- --:--:-- 20740
No architecture provided, using: amd64
Checking if all commands are available: curl jq tar
Downloading https://storage.googleapis.com/cri-o/artifacts/cri-o.amd64.v1.26.0.tar.gz to /tmp/tmp.MtcN3NZqN2
/tmp/tmp.MtcN3NZqN2 /home/****
Installing CRI-O
+ install -Z -d -m 755 /etc/cni/net.d
+ install -Z -D -m 755 -t /opt/cni/bin cni-plugins/bandwidth cni-plugins/bridge cni-plugins/dhcp cni-plugins/firewall cni-plugins/host-device cni-plugins/host-local cni-plugins/ipvlan cni-plugins/loopback cni-plugins/macvlan cni-plugins/portmap cni-plugins/ptp cni-plugins/sbr cni-plugins/static cni-plugins/tuning cni-plugins/vlan cni-plugins/vrf
+ install -Z -D -m 644 -t /etc/cni/net.d contrib/10-crio-bridge.conflist
install: cannot stat 'contrib/10-crio-bridge.conflist': No such file or directory
```

When I open the ` cri-o.amd64.v1.26.0.tar.gz` and `ls` in the `contrib/ `dir I see the following:

```
ls
11-crio-ipv4-bridge.conflist	crio.service			policy.json
```

### What did you expect to happen?

I expected the installation to go through. So either `10-crio-bridge.conflist `should be present or then the script should expect the new file which is `11-crio-ipv4-bridge.conflist`

### How can we reproduce it (as minimally and precisely as possible)?

curl https://raw.githubusercontent.com/cri-o/cri-o/v1.26.0/scripts/get | bash

### CRI-O and Kubernetes version

v1.26.0

### OS version

CentOS stream 9

### Additional environment details (AWS, VirtualBox, physical, etc.)

Opesntack

**Comment by kashifest** [comment]
I think a possible fix is here #6476

**Comment by haircommander** [comment]
thanks for the report! yes I believe #6476 will fix this

**Comment by amarlearning** [comment]
@kashifest I believe this issue can be closed now since #6482 is already merged to the release branch of CRI-O 1.26

**Comment by kashifest** [comment]
Thanks @amarlearning
